Flaky Logstash node_stats tests

Currently the Logstash node_stats test is flaky. It faily with the following error:

```
--- FAIL: TestFetch (2.74s)
	Error Trace:	node_stats_integration_test.go:21
	Error:		Received unexpected error "HTTP error 500 in node_stats: 500 Internal Server Error"
		
FAIL
```

Interestingly the `node` integration tests pass consistently without issue, meaning LS is running fine.  I wonder under which circumstances node_stats returns a 500. As it can't be reproduced manually I assume it's kind of a timing issue that `node_stats` is not ready or similar. Unfortunately there are no LS logs here for now.

This is running against Logstash master.

@elastic/logstash Any hints are appreciated here.

--- a/metricbeat/module/logstash/_meta/Dockerfile
+++ b/metricbeat/module/logstash/_meta/Dockerfile
@@ -1,4 +1,4 @@
-FROM docker.elastic.co/logstash/logstash:6.0.0-rc2
+FROM docker.elastic.co/logstash/logstash:6.0.0
 
 ENV XPACK_MONITORING_ENABLED=FALSE
-HEALTHCHECK --interval=1s --retries=90 CMD curl -f http://localhost:9600
+HEALTHCHECK --interval=1s --retries=90 CMD curl -f http://localhost:9600/_node/stats

--- a/metricbeat/module/logstash/node_stats/node_stats_integration_test.go
+++ b/metricbeat/module/logstash/node_stats/node_stats_integration_test.go
@@ -13,8 +13,6 @@ import (
 )
 
 func TestFetch(t *testing.T) {
-	t.Skipf("Test is currently skipped as `node_stats` test is flaky.")
-
 	compose.EnsureUpWithTimeout(t, 120, "logstash")
 
 	f := mbtest.NewEventFetcher(t, logstash.GetConfig("node_stats"))

